include/seahorn/Expr/Smt/Z3.hh

```diff
@@ -4,6 +4,7 @@
 
  */
 
+#include "seahorn/Expr/ExprCore.hh"
```

Comment:
why is this needed?

Reply:
removing this code for now from this PR.